template/README.md

```diff
@@ -70,6 +70,14 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 ## How To...
 
+### Install a Dependency
+
+The generated project only includes *build* dependencies. Project-specific dependencies (Redux, React Router, etc.) must be installed separately. These dependencies can be installed with `npm`, without having to modify any existing configuration.
```

Comment:
Good point about React and ReactDOM also being dependencies. I like the explicitness of your copy, thanks for the suggestion!